util/bump_version.sh

```diff
@@ -16,7 +16,50 @@ prompt_for_number() {
 	done
 }
 
+# On a release the following actions are performed
+# * DEVELOPMENT_BUILD is set to false
+# * android versionCode is bumped
+# * Commit the changes
+# * Tag with current version
+perform_release() {
+	sed -i -re "s/^set\(DEVELOPMENT_BUILD TRUE\)$/set(DEVELOPMENT_BUILD FALSE)/" CMakeLists.txt || die "Failed to unset DEVELOPMENT_BUILD"
 
+	sed -i -re "s/versionCode [0-9]+$/versionCode $NEW_ANDROID_VERSION_CODE/" build/android/build.gradle || die "Failed to update Android version code"
+
+	git add -f CMakeLists.txt build/android/build.gradle || die "git add failed"
+
+	git commit -m "Bump version to $RELEASE_VERSION" || die "git commit failed"
+
+	echo "Tagging $RELEASE_VERSION"
+
+	git tag -a "$RELEASE_VERSION" -m "$RELEASE_VERSION" || die 'Adding tag failed'
+}
+
+# After release
+# * Set DEVELOPMENT_BUILD to true
+# * Bump version in CMakeLists and docs
+# * Commit the changes
+back_to_devel() {
+	echo 'Creating "return back to development" commit'
+
+	sed -i -re 's/^set\(DEVELOPMENT_BUILD FALSE\)$/set(DEVELOPMENT_BUILD TRUE)/' CMakeLists.txt || die 'Failed to set DEVELOPMENT_BUILD'
+
+	sed -i -re "s/^set\(VERSION_MAJOR [0-9]+\)$/set(VERSION_MAJOR $NEXT_VERSION_MAJOR)/" CMakeLists.txt || die "Failed to update VERSION_MAJOR"
+
+	sed -i -re "s/^set\(VERSION_MINOR [0-9]+\)$/set(VERSION_MINOR $NEXT_VERSION_MINOR)/" CMakeLists.txt || die "Failed to update VERSION_MINOR"
+
+	sed -i -re "s/^set\(VERSION_PATCH [0-9]+\)$/set(VERSION_PATCH $NEXT_VERSION_PATCH)/" CMakeLists.txt || die "Failed to update VERSION_PATCH"
+
+	sed -i -re "1s/[0-9]+\.[0-9]+\.[0-9]+/$NEXT_VERSION/g" doc/lua_api.txt || die "Failed to update doc/lua_api.txt"
+
+	sed -i -re "1s/[0-9]+\.[0-9]+\.[0-9]+/$NEXT_VERSION/g" doc/menu_lua_api.txt || die "Failed to update doc/menu_lua_api.txt"
+
+	sed -i -re "1s/[0-9]+\.[0-9]+\.[0-9]+/$NEXT_VERSION/g" doc/client_lua_api.md || die "Failed to update doc/client_lua_api.md"
```

Comment:
the problem when splitting a such line in shell is making the command unreadable :s